utils/ErrorOutput.go

```diff
@@ -0,0 +1,45 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"log"
+	"net/http"
+)
+
+type ValidationError struct {
+	Field string
+	Error string
+}
+
+func ClientError(status int, body string) (events.APIGatewayProxyResponse, error) {
+
+	resp := events.APIGatewayProxyResponse{
+		Body:       body,
+		StatusCode: status,
+		Headers: map[string]string{
+			"Content-Type": "application/json",
+		},
+	}
+
+	return resp, nil
+}
+func ServerError(err error) (events.APIGatewayProxyResponse, error) {
+	errMsg := ""
+	if err != nil {
+		errMsg = err.Error()
+	}
+	return events.APIGatewayProxyResponse{
+		Body:       errMsg,
+		StatusCode: http.StatusInternalServerError,
+	}, fmt.Errorf("Internal server error: %v", err)
```

Comment:
It's not recommended to expose internal error messages. So just passing a generic error message with a 500 status code and logging the error seems a better alternative

Reply:
Will we be doing that change in this PR?
IMO, I think we should